packages/manager/apps/dedicated/client/app/components/dedicated-cloud/dedicatedCloud.service.js

```diff
@@ -536,6 +536,47 @@ class DedicatedCloudService {
     });
   }
 
+  /* ------- IAM -------*/
+
+  getIamStatus(serviceName) {
+    return this.$http
+      .get(`/dedicatedCloud/${serviceName}/iam`)
+      .then(({ data }) => data);
+  }
+
+  isIamTogglable(serviceName, state, _returnAsBoolean = true) {
+    const returnAsBoolean = _returnAsBoolean;
+
+    const endpoint = `canBe${state === 'enabled' ? 'Disabled' : 'Enabled'}`;
+
+    return this.$http
+      .get(`/dedicatedCloud/${serviceName}/iam/${endpoint}`)
+      .then(({ data }) => {
+        // API return a funny response, assume it is true if the response was a 200
+        return returnAsBoolean ? { toggable: true } : data;
+      });
```

Comment:
```suggestion
      .then(() => true)
```
https://github.com/ovh/manager/pull/9649#discussion_r1277304830